src/navigation/wallet/screens/send/SendTo.tsx

```diff
@@ -522,6 +522,7 @@ const SendTo = () => {
         keyId,
         address,
         currency: credentials.coin,
+        chain: wallet.chain,
```

Comment:
I think it's better to use selectedWallet `chain` value

Reply:
Fixed. Thanks!